plugin/xcodebuild.vim

```diff
@@ -3,13 +3,27 @@ command! XTest call <sid>test()
 
 let s:plugin_path = expand('<sfile>:p:h:h')
 
+function! s:bin_script(name)
+  return s:plugin_path . '/bin/' . a:name
+endfunction
+
+function! s:cli_args(...)
+  let cli_args = ''
+
+  for cli_arg in a:000
```

Comment:
So, this is kind of crazy, but I think you can replace this whole function with this per `:help map()`:

```vim
let cli_args = join(map(a:000, shellescape(v:val)))
```